internal/api/api.go

```diff
@@ -71,13 +73,13 @@ func New(engine *opa.OPA) *API {
 }
 
 // Run starts the HTTP server.
-func (api *API) Run(ctx context.Context) error {
+func (api *API) Run(_ context.Context) error {
```

Comment:
[nit] We're the only callers. If we don't care for the context argument, we could drop it.

Reply:
Yeah, I basically just fixed some linter warnings along the way.. someone could definitely do some more thorough touch ups here and there.. I'll let this be for now though.